cluster/1.0.1/kubevirt-ssp-operator-cr.yaml

```diff
@@ -0,0 +1,22 @@
+---
+apiVersion: kubevirt.io/v1
+kind: KubevirtCommonTemplatesBundle
+metadata:
+  name: kubevirt-common-template-bundle
+spec:
+  version: v0.4.1
```

Comment:
Hmm, how come the version here was not bumped to 0.6.0?

Reply:
right, fixed